xdp-forward/xdp-forward.c

```diff
@@ -217,6 +222,34 @@ static int do_load(const void *cfg, __unused const char *pin_root_path)
 		pr_info("Loaded on interface %s\n", iface->ifname);
 	}
 
+	/* Attach xdp_check_return to xdp_frame_return tracepoint and
+	 * pin the link to pin_root_path/xdp_check_return.
+	 */
+
+	return_prog = bpf_object__find_program_by_name(skel->obj,"xdp_check_return");
+	if (!return_prog) {
+		pr_warn("Failed to find xdp_check_return program\n");
+		goto end_detach;
+	}
+
+	link = bpf_program__attach_raw_tracepoint(return_prog, "xdp_frame_return");
+	if (libbpf_get_error((const void *) link)) {
```

Comment:
It should be possible to just pass the pointers to `libbpf_get_error()` without the cast...